config/scripts/patch-ts.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs')
+const path = require('path')
+const { replaceCoreImports } = require('../replaceCoreImports')
+
+async function patchTypeDefs() {
```

Comment:
Type defs emitted by TS still have the `~/core` imports in them. Nothing resolves them to actual `paths` defined in tsconfig. Well, someone must.